cmd/clusterctl/client/cluster/mover.go

```diff
@@ -60,15 +60,15 @@ func (o *objectMover) Move(namespace string, toCluster Client) error {
 	}
 
 	// Gets all the types defines by the CRDs installed by clusterctl plus the ConfigMap/Secret core types.
-	types, err := objectGraph.getDiscoveryTypes()
+	err := objectGraph.getDiscoveryTypes()
 	if err != nil {
 		return err
 	}
 
 	// Discovery the object graph for the selected types:
 	// - Nodes are defined the Kubernetes objects (Clusters, Machines etc.) identified during the discovery process.
 	// - Edges are derived by the OwnerReferences between nodes.
-	if err := objectGraph.Discovery(namespace, types); err != nil {
+	if err := objectGraph.Discovery(namespace); err != nil {
```

Comment:
Might be a topic for a follow-up PR/something to be tracked in a issue, but once we determine the external CRD to move, it would be great to check if they exist on the target cluster (we are already doing this for the providers in checkTargetProviders)